test/Invio.QueryProvider.Test.CSharp/QueryableTestBase.cs

```diff
@@ -476,6 +488,35 @@ public abstract class QueryableTestBase {
             );
         }
 
+        private static String[] CustomerIdsTitleOwnerAssistant { get; } =
+            { "WILMK" };
+
+        [Fact]
+        public virtual void Where_String_Equals() {
+            var results =
+                this.Customers
+                    .Where(c => c.ContactTitle.Equals("Owner/Marketing Assistant"))
+                    .ToList();
+
+            AssertCustomerIdSet(
+                CustomerIdsTitleOwnerAssistant,
+                results
+            );
+        }
+
+        [Fact]
+        public virtual void Where_String_StaticEquals() {
+            var results =
+                this.Customers
+                    .Where(c => String.Equals(c.ContactTitle, "Owner/Marketing Assistant"))
```

Comment:
It's good that you are testing forward slashes in addition to alphanumeric characters here. The sample data [has a lot of test data with accents](https://github.com/invio/Invio.QueryProvider.Core/blob/master/test/Invio.QueryProvider.Test/Data/Customer.csv) that would good test cases as well.